src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { validateText, validateToCase } = require('./validateData');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normUrl = new URL(req.url, 'http://localhost:5700');
+    const text = normUrl.pathname.slice(1);
+    const toCase = normUrl.searchParams.get('toCase');
+    const errorsArray = [];
+
+    const textError = validateText(text);
+    const toCaseError = validateToCase(toCase);
+
+    if (textError) {
+      errorsArray.push(textError);
+    }
+
+    if (toCaseError) {
+      errorsArray.push(toCaseError);
+    }
```

Comment:
Create a function that will validate data and move it to a separate file, I meant all validations do in this function instead